modules/cnf-running-the-performance-creator-profile-offline.adoc

```diff
@@ -0,0 +1,228 @@
+  [ -f "${MG_TARBALL}" ] || exit_error "Must-gather tarball file not found"
+
+  DATA_DIR=$(mktemp -d -t "${CURRENT_SCRIPT}XXXX") || exit_error "Cannot create the data directory"
+  tar -zxf "${MG_TARBALL}" --directory "${DATA_DIR}" || exit_error "Cannot decompress the must-gather tarball"
+  chmod a+rx "${DATA_DIR}"
+
+  return 0
+}
+
+main() {
+  while getopts ':hp:t:' OPT; do
+    case "${OPT}" in
+      h)
+        usage
+        exit 0
+        ;;
+      p)
+        PAO_IMG="${OPTARG}"
+        ;;
+      t)
+        MG_TARBALL="${OPTARG}"
+        ;;
+      ?)
+        exit_error "invalid argument: ${OPTARG}"
+        ;;
+    esac
+  done
+  shift $((OPTIND - 1))
+
+  check_requirements || exit 1
+
+  ${CMD} -v "${DATA_DIR}:${MUST_GATHER_VOL}:z" "${PAO_IMG}" "$@" --must-gather-dir-path "${MUST_GATHER_VOL}"
+  echo "" 1>&2
+}
+
+main "$@"
+----
+
+. Add execute permissions for everyone on this script:
++
+[source,terminal]
+----
+$ chmod a+x run-perf-profile-creator.sh
+----
+
+. (Optional) Display the `run-perf-profile-creator.sh` command usage:
++
+[source,terminal]
+----
+$ ./run-perf-profile-creator.sh -h
+----
+.Expected output
++
+[source,terminal]
+----
+Wrapper usage:
+  run-perf-profile-creator.sh [-h] [-p image][-t path] -- [performance-profile-creator flags]
+
+Options:
+   -h                 help for run-perf-profile-creator.sh
+   -p                 Performance Addon Operator image <1>
+   -t                 path to a must-gather tarball
+A tool that automates creation of Performance Profiles
+
+Usage:
+  performance-profile-creator [flags]
+
+Flags:
+      --disable-ht                        Disable Hyperthreading
+  -h, --help                              help for performance-profile-creator
+      --mcp-name string                   MCP name corresponding to the target machines (required) (default "worker-cnf")
+      --must-gather-dir-path string       Must gather directory path (default "must-gather")
+      --power-consumption-mode string     The power consumption mode.  [Valid values: default, low-latency, ultra-low-latency] (default "default")
+      --profile-name string               Name of the performance profile to be created (default "performance")
+      --reserved-cpu-count int            Number of reserved CPUs (required)
+      --rt-kernel                         Enable Real Time Kernel (required) (default true)
+      --split-reserved-cpus-across-numa   Split the Reserved CPUs across NUMA nodes
+      --topology-manager-policy string    Kubelet Topology Manager Policy of the performance profile to be created. [Valid values: single-numa-node, best-effort, restricted] (default "restricted")
+      --user-level-networking             Run with User level Networking(DPDK) enabled
+----
++
+<1> Specify the Performance Addon Operator image, if nothing is set the default upstream image is used namely `quay.io/openshift-kni/performance-addon-operator:4.8-snapshot`.
```

Comment:
I think there is room to better explain the  wrapper arguments -p the image as appears above, and -t the must-gather as a tarball